godot-kotlin/godot-library/src/nativeGen/.gitignore

```diff
@@ -0,0 +1 @@
+kotlin/*
```

Comment:
this is to ignore the generated sources right?

Reply:
yep